projectcard/schema/changes/transit_service_deletion.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+	"required": [
+		"service",
+        "clean_shapes",
+        "clean_routes"
+	],
+	"additionalProperties" : false,
+	"properties": {
+		"service": {"$ref": "../transit/selection/select_trips.json"},
+        "clean_shapes": {
+            "type": "boolean",
+            "description": "If true, unused shapes for these trips will be removed. Defaults to true.",
+            "default": true
+        },
```

Comment:
@i-am-sijia we will need to set `clean_shape: false` for all the metcoucil transit project cards since removing unused shapes will change the order of shape_index we assigned to each shape.

Reply:
I see. Is `clean_shape: false` needed for all project cards or just the transit deletion project cards? We could force Lasso to write out false, correct?

Reply:
yes, I'll modify the code to write out false. I'm thinking to add it for all the transit project cards or at least transit deletion and property change. Changing the routing of a route for all time periods could also generate an unused shape.

Reply:
> I can also make the default false if that is easier (for this and route). Up to you all - i don't have strong feelings about it.

Will clean_shape parameter affect any transit routing change? i.e. changing the routing of a route for all time periods will generate a new shape and an unused shape.